src/task/CreatePeerConnectionTask.ts

```diff
@@ -187,7 +199,31 @@ export default class CreatePeerConnectionTask extends BaseTask implements Remova
       width = cap.width as number;
       height = cap.height as number;
     }
-    tile.bindVideoStream(attendeeId, false, stream, width, height, streamId);
+    const externalUserId = this.context.realtimeController.realtimeExternalUserIdFromAttendeeId(
+      attendeeId
+    );
+    tile.bindVideoStream(attendeeId, false, stream, width, height, streamId, externalUserId);
+    if (!externalUserId) {
```

Comment:
Suggest moving the contents of this if-body into a separate function with a name that indicates what it is doing.